chalice/deploy/models.py

```diff
@@ -289,6 +289,12 @@ def dependencies(self):
         return cast(List[Model], resources)
 
 
+@attrs
+class Endpoint(Model):
+    type = attrib()                         # type: str
+    vpce_ids = attrib(default=None)         # type: Union[str, List[str]]
```

Comment:
(in reference to my comment in `appgraph.py`) I think we can simplify the code and types if we just require this to be `List[str]` and if a user provides a string in their config file just convert it to a list of a single string.